incubator/oauth-proxy/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.config.configFile }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "oauth-proxy.name" . }}
```

Comment:
`oauth2-proxy.name`

Reply:
I've taken care of this in PR: https://github.com/kubernetes/charts/pull/4744